Gopkg.toml

```diff
@@ -28,7 +28,8 @@ required = [
 
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  branch = "iss-1137"
```

Comment:
/hold

WIP

Reply:
Rebased with master, no longer uses gopkg.toml. The `knative.dev/pkg` changes we needed are also merged into master of serving. Updated PR to only have relevant change.